lib/colorls/flags.rb

```diff
@@ -49,16 +53,27 @@ def add_sort_options(options)
       options.separator ''
       options.separator 'sorting options:'
       options.separator ''
-      options.on('--sd', '--sort-dirs', '--group-directories-first', 'sort directories first') { @opts[:sort] = :dirs }
-      options.on('--sf', '--sort-files', 'sort files first')                                   { @opts[:sort] = :files }
+      options.on('--sd', '--sort-dirs', '--group-directories-first', 'sort directories first') { @opts[:group] = :dirs }
+      options.on('--sf', '--sort-files', 'sort files first')                                  { @opts[:group] = :files }
+      options.on('-t', 'sort by modification time, newest first')                             { @opts[:sort] = :time }
+      options.on('-S', 'sort by file size, largest first')                                    { @opts[:sort] = :size }
+      options.on('--sort=WORD', %w[none time size], 'sort by WORD instead of name: none, size (-S), time (-t)') do |word|
+        @opts[:sort] = case word
+                       when 'none' then false
+                       when 'time' then :time
+                       when 'size' then :size
+                       end
+      end
+
+      options.on('-r', '--reverse', 'reverse order while sorting') { @opts[:reverse] = true }
     end
 
     def add_common_options(options)
       options.on('-a', '--all', 'do not ignore entries starting with .')  { @opts[:all] = true }
       options.on('-A', '--almost-all', 'do not list . and ..')            { @opts[:almost_all] = true }
       options.on('-l', '--long', 'use a long listing format')             { @mode = :long }
       options.on('-t', '--tree', 'shows tree view of the directory')      { @mode = :tree }
```

Comment:
The `-t` for `--tree` flag has to be removed, and same to be mentioned in the `POST_INSTALL_MESSAGE`

Reply:
Sure! I changed that and pushed the changes.